src/protections/ProtectionManager.ts

```diff
@@ -332,7 +332,18 @@ export class ProtectionManager {
 
     private async handleEvent(roomId: string, event: any) {
         if (this.mjolnir.protectedRoomsTracker.getProtectedRooms().includes(roomId)) {
-            if (event["sender"] === (await this.mjolnir.client.getUserId())) return; // Ignore ourselves
+            const sender = event["sender"];
+            if (sender === (await this.mjolnir.client.getUserId())) return; // Ignore ourselves
+            // we don't want to apply protection actions to members of the moderation room, thus don't
+            // need to spend the cycles checking the event
+            if (this.mjolnir.moderators.checkMembership(sender)) {
+                this.mjolnir.managementRoomOutput.logMessage(
+                    LogLevel.WARN,
+                    "Protection",
+                    `Skipping checking event from ${sender} in protections`,
+                );
```

Comment:
this will probably get noisy very quickly - every message sent by an admin will result in a warning in the management room. For us it's only ~100 active rooms, but for some communities this is easily over a thousand rooms.

The log line feels important, but probably doesn't need to be brought into the management room. Just the log file/console is fine.